api/design/resource.go

```diff
@@ -121,7 +121,7 @@ var _ = Service("resource", func() {
 				Example("catalog", "tektoncd")
 			})
 			Attribute("kind", String, "kind of resource", func() {
-				Enum("task", "pipeline")
+				Enum("task", "pipeline", "Task", "Pipeline")
```

Comment:
I think hub api service should only allow `lower case`. It should be responsibility of the cli tool / client tools to convert from `Pipeline` to `pipeline`

Reply:
@sthaha 
Case 1 :
`/resource/Tekton/Task/Buildah` returns an error because `Task` should be `task`.  catalog name and resource name can be in title case
Case 2:
all our API responses have `kind`: Task Pipeline in the title case but API doesn't allow input kind as Title case.
due to which decided to allow title case. WDYT?
should the response be in lower case?